terraso_backend/apps/soilproj/admin.py

```diff
@@ -0,0 +1,17 @@
+# Copyright © 2021-2023 Technology Matters
```

Comment:
Suggestion: The name `soilproj` is not understandable, can we have a complete name here